gui/wxpython/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+
+# missing docs
+
+
+set(WXPYTHON_DIR ${GISBASE}/gui/wxpython)
+set(gui_lib_DIRS core gui_core icons iscatt lmgr location_wizard
+mapdisp mapwin modules nviz rdigit startup tools vnet web_services wxplot)
+
+set(gui_lib_targets)
+foreach(gui_lib_DIR ${gui_lib_DIRS})
+  copy_python_files_in_subdir(${gui_lib_DIR} gui/wxpython)
+  list(APPEND gui_lib_targets python_${gui_lib_DIR})
+endforeach()
+add_custom_target(GUI_WXPYTHON
+	COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/gis_set.py ${WXPYTHON_DIR}/
+	COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/gis_set_error.py ${WXPYTHON_DIR}/
```

Comment:
These files were removed in PR #1400. The makefile `gui/wxpython/Makefile` in this diff simply removed the two files in the copy list, so I removed them here too https://github.com/OSGeo/grass/commit/8fe13636b7081332b09d5b75faa2dac3fa015bef#diff-ada932711263adc330ae9a860f27fb8b90ba8285747d0effd05661359660fd03
```suggestion
```